pyfar/dsp/filter/audiofilter.py

```diff
@@ -224,25 +226,132 @@ def bell(signal, center_frequency, gain, quality, bell_type='II',
         return signal_filt
 
 
-def high_shelve(signal, frequency, gain, order, shelve_type='I',
+@rename_arg({"shelve_type": "shelf_type"}, "shelve_type will be deprecated in"
+            " version 0.8.0 in favor of shelf_type")
+def high_shelve(signal, frequency, gain, order, shelf_type='I',
```

Comment:
I don't think we need to deprecate parameters in a deprecated function, it just confuses people. I would leave it as it is and just deprecate it and add the new one.

Reply:
But we should properly deprecate "shelve_type" at some point or is it ok to rename the parameter if the function is renamed?

Reply:
Yes I think you can deprecate the old method and introduce the new one, directly with the new parameter name. If the old one has a "wrong" name it does not matter, its deprecated anymay trough the function.